.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    ["es2015"],
```

Comment:
`es2015` has been depreciated. Please use [babel-preset-latest](https://babeljs.io/docs/en/babel-preset-latest) instead.

Reply:
Or we can use [babel-preset-env](https://github.com/babel/babel-preset-env)